contrib/kafka/Readme.md

```diff
@@ -0,0 +1,103 @@
+# Knative KafkaEventSource
+
+This deploys an Operator that can instantiate Knative Event Sources which subscribe to a Kafka Topic.
+
+## EventSource
+
+The image that receives messages from Kafka (Receive Adaptor) is available at `docker.io/sjwoodman/kafkaeventsource`.
```

Comment:
Found this container registry, guessing official images are hosted there. https://gcr.io/knative-releases/github.com/knative/eventing-sources